packages/discord.js/src/util/Transformers.js

```diff
@@ -33,4 +34,25 @@ function _transformAPIAutoModerationAction(autoModerationAction) {
   };
 }
 
-module.exports = { toSnakeCase, _transformAPIAutoModerationAction };
+/**
+ * Transforms an API integration types config object to a camel-cased variant.
+ * @param {APIApplicationIntegrationTypeConfiguration} integrationTypesConfiguration The data to transform
+ * @returns {IntegrationTypesConfiguration}
+ * @ignore
+ */
+function _transformAPIIntegrationTypesConfiguration(integrationTypesConfiguration) {
+  console.log(integrationTypesConfiguration);
+  return integrationTypesConfiguration
+    ? {
+        oAuth2InstallParams:
+          {
+            scopes: integrationTypesConfiguration.oauth2_install_params?.scopes ?? null,
+            permissions: integrationTypesConfiguration.oauth2_install_params
+              ? new PermissionsBitField(integrationTypesConfiguration.oauth2_install_params.permissions).freeze()
+              : null,
+          } ?? null,
```

Comment:
This `?? null` is useless, you always define an object here
```suggestion
          },
```